### DISCUSSION TITLE
"Requested entity is not found" and "The task no longer exists" errors

### DISCUSSION THREAD

**Discussion by i386** [stackkit/laravel-google-cloud-tasks-queue#89] [General]
I am receiving a lot of errors being reported via error reporting relating to Google Tasks.

Here are a few common ones:
* "Requested entity is not found"
* "The task no longer exists, though a task with this name existed recently. The task either successfully completed or was deleted" errors

Unfortunately the sources of these errors seem to be hidden as they are thrown within promises of the Google Tasks library. Has anyone experienced this? 

<img width="1634" alt="image" src="https://user-images.githubusercontent.com/50156/206364218-2e21e26a-bb7e-45f2-8b01-9c93fde1c5bc.png">

@marickvantuil

**Comment by marickvantuil** [comment]
@i386 We're trying to fix this issue in #95 and wondering if you can join that discussion and share some details or extra info which we can use to find the problem? Thanks!

**Reply by i386** [reply]
Yeah add me to this. I tried contacting paid google support as well (that error message is super bad) but they were useless.
